cmd/stork/stork.go

```diff
@@ -59,16 +59,17 @@ import (
 )
 
 const (
-	defaultLockObjectName      = "stork"
-	defaultLockObjectNamespace = "kube-system"
-	defaultAdminNamespace      = "kube-system"
-	storkVersion               = "version"
-	cmName                     = "stork-version"
-	eventComponentName         = "stork"
-	debugFilePath              = "/var/cores"
-	awsKopiaExecutorImage      = "709825985650.dkr.ecr.us-east-1.amazonaws.com/portworx/kopiaexecutor"
-	awsKopiaExecutorImageTag   = "1.0.0-a345bb2"
-	awsMarketPlace             = "aws"
+	defaultLockObjectName       = "stork"
+	defaultLockObjectNamespace  = "kube-system"
+	defaultAdminNamespace       = "kube-system"
+	storkVersion                = "version"
+	cmName                      = "stork-version"
+	eventComponentName          = "stork"
+	debugFilePath               = "/var/cores"
+	awsKopiaExecutorImage       = "709825985650.dkr.ecr.us-east-1.amazonaws.com/portworx/kopiaexecutor"
+	awsKopiaExecutorImageTag    = "1.0.0-a345bb2"
+	awsMarketPlace              = "aws"
+	defaultControllerResyncTime = 90 * time.Second
```

Comment:
this will change rsync interval for all controllers, you can pass in same to kdmp controller as well https://github.com/libopenstorage/stork/blob/master/pkg/applicationmanager/controllers/applicationbackup.go#L111